core/templates/dev/head/player/PlayerServices.js

```diff
@@ -692,3 +748,53 @@ oppia.directive('feedbackPopup', ['oppiaPlayerService', function(oppiaPlayerServ
     }]
   };
 }]);
+
+oppia.controller('InformationCard', function ($scope, $modal) {
+
+  $scope.showInformationCard = function () {
+
+    var modalInstance = $modal.open({
+      animation: true,
+      templateUrl: 'popover/informationCard',
+      windowClass: 'oppia-modal-information-card',
+      controller: ['$scope', '$http', '$modal', '$modalInstance','oppiaPlayerService', 'ratingService',
```

Comment:
nit: add space after comma (after $modalInstance)

Can you make these lines < 100 chars?